cmd/sealos/cmd/merge.go

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2021 sealos.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/labring/sealos/pkg/image"
+	"github.com/labring/sealos/pkg/image/types"
+	"github.com/labring/sealos/pkg/utils/logger"
+)
+
+func newMergeCmd() *cobra.Command {
+	var options types.BuildOptions
+	var newImageName string
+	var mergeCmd = &cobra.Command{
+		Use:   "merge",
+		Short: "merge multiple images into one",
+		Long:  `sealos merge image1:latest image2:latest image3:latest ......`,
+		Example: `
+merge images:
+	sealos merge kubernetes:v1.19.9 mysql:5.7.0 redis:6.0.0 -t new:0.1.0
```

Comment:
`sealos merge x1 x2 x3 x4`, `x4` is the final image name.

Reply:
Specifying parameters may be clearer.